web/rootfs/etc/cont-init.d/10-config

```diff
@@ -10,24 +10,37 @@ mkdir -p \
 # generate keys (maybe)
 if [[ $DISABLE_HTTPS -ne 1 ]]; then
     if [[ $ENABLE_LETSENCRYPT -eq 1 ]]; then
-        mkdir -p /config/acme.sh
-        pushd /opt
-        sh ./acme.sh --install --home /config/acme.sh --accountemail $LETSENCRYPT_EMAIL
-        popd
+        mkdir -p /config
+        cd /config/
+        git clone git@github.com:acmesh-official/acme.sh.git
+        git checkout 3.0.6
+        cd acme.sh
+        sh ./acme.sh --install \
+            --home /config/acme.sh \
+            --accountemail $LETSENCRYPT_EMAIL \
+            --register-account \
+            --server zerossl
 
-        STAGING=""
-        if [[ $LETSENCRYPT_USE_STAGING -eq 1 ]]; then
-            STAGING="--staging"
+        if [ ! -z "$CF_Token" ]; then
```

Comment:
Can't you combine these 2 into a single one by having some VALIDATION variable which would be `--standalone` or `--dns dns_cf` ?